main.css

```diff
@@ -363,3 +371,405 @@ form button {
   border: 1px solid #cfd8dc;
   margin-bottom: 45px;
 }
+
+@media screen and (min-width: 800px) {
```

Comment:
Use media queries in your CSS file to make your desktop version look like the template you selected (let's assume that the desktop width should be in the 992px - 1100px range).